tzrec/datasets/data_parser.py

```diff
@@ -423,6 +423,8 @@ def _to_sparse_features(
                 weights=torch.cat(weights, dim=-1)
                 if len(dg_has_weight_keys) > 0
                 else None,
+                stride=lengths[0].size(0),  # input_data['input_batch_size']
+                length_per_key=[x.numel() for x in values],
```

Comment:
for loop may be slow?

Reply:
iterate over features, not over batches, the cost is relatively small